src/TLuaInterpreter.cpp

```diff
@@ -3976,11 +3976,14 @@ int TLuaInterpreter::connectToServer( lua_State *L )
 {
     int port;
     string url;
+    bool saveToProfile = false;
+
+    Host * pHost = TLuaInterpreter::luaInterpreterMap[L];
+
     if( ! lua_isstring( L, 1 ) )
     {
-        lua_pushstring( L, "connectToServer: wrong argument type" );
-        lua_error( L );
-        return 1;
+        lua_pushstring( L, QString(" connectToServer: argument #1 expects type string, %1 was given.").arg( lua_typename(L,1) ).toUtf8().constData() );
```

Comment:
The space was habitual, sorry.  The way it gets printed on the debug console is missing a space between the start of the functions error text and the Lua generic error text that wraps it.  I was compelled to add the space.  haha.